src/odoc/etc/odoc.css

```diff
@@ -406,6 +406,9 @@ pre code {
   padding: 0.35em 0.5em;
 }
 
+.spec .def-doc { user-select:none; webkit-user-select:none }
+.spec .def-doc:hover { user-select:auto; webkit-user-select:auto }
+
```

Comment:
I highly dislike this. In my experience, websites that mess with copy paste and selection are the leading cause of internet rage and deserve to burn in hell. You are free to use it in your CSS, but I really don't want that in the default one.

However, I got an idea: what about invisible `(*`/`*)` delimiters, which would no take visual space, but would get selected during copy pasting ? I'm pretty sure we can cook something up with some careful styling.

Reply:
> I highly dislike this. In my experience, websites that mess with copy paste and selection are the leading cause of internet rage and deserve to burn in hell.

That seems a little bit excessive. Did you actually [try it](https://b0-system.github.io/odig/doc/biniou/Bi_inbuf/index.html) ? 

Until something better is found I think we should leave it, it certainly provides a better user experience for copy pasting type definitions. 

I don't mind removing that commit if other people agree with you, but honestly the result provided by default stylesheet is both unreadable and ergonomically lacking, I'm offering improvements but I'm not interested in fighting for them.

> However, I got an idea: what about invisible `(*`/`*)` delimiters, which would no take visual space,

You can try to do this. Another thing that could be tried is to add the comment delimiters with CSS pseudo selectors.

Reply:
> You can try to do this. Another thing that could be tried is to add the comment delimiters with CSS pseudo selectors.

I just did! See https://github.com/ocaml/odoc/pull/626, I think it works quite well.